core/state/snapshot/snapshot.go

```diff
@@ -173,7 +173,7 @@ type Tree struct {
 // and will be reconstructed from scratch based on the tries in the key-value
 // store, on a background thread. If the memory layers from the journal is not
 // continuous with disk layer or the journal is missing, all diffs will be discarded
-// iff it's in "recovery" mode, otherwise rebuild is mandatory.
+// if it's in "recovery" mode, otherwise rebuild is mandatory.
```

Comment:
The word `iff` is usually an abbreviation of `If and only if`. Not 100% sure if that's what the author intended here, but might be

Reply:
Oh, that makes sense.